index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CodeJam1-CV</title>
+    <link rel="stylesheet" href="https://meyerweb.com/eric/tools/css/reset/reset.css">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css"
+    integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">
+</head>
+
+<body>
+
+    <div class="wrapper">
+      <header class="header">
+
+          <img class="photo" src="images/photo.png" alt="photo" >
+
+
+        <h1>Maksim Kalachou</h1>
+        <h2>Junior JS Dev</h2>
+        <p class="contacts">
+        Minsk/Mogilev, Belarus<br>
+        <i class="fas fa-phone-volume"></i>
```

Comment:
`<i>` tags are located inside` <p class="contacts">` tag, so `<i>` tags should be shifted right

Example:
```
<p class="contacts">
      <i class="fas fa-phone-volume"></i>
      <i class="fas fa-phone-volume"></i>
</p>
```
Always check indents